apps/admin-portal/src/components/users/wizard/add-user-wizard.tsx

```diff
@@ -383,11 +396,16 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
      */
     const addUserBasic = (userInfo: any) => {
         let userName = "";
+        let profileUrl = "";
         userInfo.domain !== "primary" ? userName = userInfo.domain + "/" + userInfo.userName : userName =
             userInfo.userName;
         let userDetails = {};
         const password = userInfo.newPassword;
```

Comment:
Shall we add the types of each `let` variable in the above lines?

Reply:
The type of a variable is inferred when we assign it with a value right? Do we need to explicitly type it?

Reply:
Yes for primitive types, but IMO for complex variables such as line `388` the type should be mentioned to reduce possible bugs. Also since the `password` is assigned from `userInfo` which is an `any` type object, specifying the type for the `password` variable will let you identify type errors before hand.

Reply:
Resolved with 8d963ab